roles/devscripts/tasks/03_install.yml

```diff
@@ -25,6 +25,17 @@
     - bootstrap
   ansible.builtin.import_tasks: sub_tasks/32_params.yml
 
+- name: Gather the metal3-dev-env repo directory information
+  ansible.builtin.stat:
+    path: "{{ cifmw_devscripts_config_defaults.working_dir }}/metal3-dev-env"
+  register: metal3_dev_env_repodir
+
+- name: Overwrite cifmw_devscripts_make_target value
+  when:
+    - metal3_dev_env_repodir.stat.exists
+  ansible.builtin.set_fact:
+    cifmw_devscripts_make_target: redeploy
```

Comment:
There is very high probability that we would not hit this code because of the precheck occurring earlier.

https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/devscripts/tasks/00_pre_check.yml

If there exists a working cluster, we would skip this stage. If there is a non-working cluster deployment, cleanup is executed within the pre-check tasks.

Reply:
redeploy method was ditched